include/expire.php

```diff
@@ -9,14 +9,32 @@ function expire_run(&$argv, &$argc){
 	require_once('include/items.php');
 	require_once('include/Contact.php');
 
+	load_hooks();
+
+	if (($argc == 2) && (intval($argv[1]) > 0)) {
```

Comment:
you changed many `intval($x)` calls to `(int)$x` but used intval here - intentionally?